hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/infoschema/MappingsTable.java

```diff
@@ -79,13 +87,27 @@ public MappingsTable(
     protected List<Object[]> rows() {
         List<Object[]> rows = new ArrayList<>(mappings.size());
         for (Mapping mapping : mappings) {
+            Map<String, String> opts = mapping.options();
+            if (securityEnabled) {
+                opts = new HashMap<>(opts);
+                SqlConnector sqlConnector = sqlConnectorCache.forType(mapping.connectorType());
+                Set<String> secureConnectorOptions = sqlConnector.secureConnectorOptions();
+                for (String key : mapping.options().keySet()) {
```

Comment:
nit: what is cheaper - copy hashmap and then remove items or only add items that are safe?